connect/src/routes/tokenBridge/automatic.ts

```diff
@@ -133,18 +125,46 @@ export class AutomaticTokenBridgeRoute<N extends Network> extends AutomaticRoute
     yield* TokenTransfer.track(this.wh, receipt, timeout, this.fromChain, this.toChain);
   }
 
-  private toTransferDetails(
-    amount: bigint,
-    options: AutomaticTokenBridgeRoute.Options,
-  ): TokenTransferDetails {
+  private async stringToBigInt(amount: string = "0"): Promise<bigint> {
```

Comment:
Instead of doing this here, lets grab the token details (name, decimals, ...) right before searching for routes and then cache that in the Route obj 

Then we don't have to make converter async